### ISSUE TITLE
expose custom logger

### ISSUE THREAD

**Issue by taylorchu** [go-mysql-org/go-mysql#170]
In canal, the log output is not structured.

In https://github.com/siddontang/go-mysql/pull/167, the idea of custom logger is briefly discussed. We would like to use logger from [logrus](https://github.com/sirupsen/logrus) for example to be able to post-process structured logs easily.

**Comment by siddontang** [comment]
Yes, @taylorchu 

Can you send me a PR? Maybe we can pass the logrus Logger to go-mysql. if not set, we can use the default standard logger.

**Comment by dveeden** [comment]
With #993 merged I think we should probably also close this now that we use `log/slog`.

@serprex @lance6716 do you agree?

**Comment by lance6716** [comment]
I agree that we can close this issue. Please refer to the Logging section in README
